doc/faq.md

```diff
@@ -136,7 +136,7 @@ const map = new Map({
 Here `[-77.036667, 38.895]` is provided as the center of the view. But as Web
 Mercator is a metric projection, you are currently telling OpenLayers that the
 center shall be some meters (~77m and ~39m respectively) away from `[0, 0]`. In
-the Web Mercator projection the coordinate is right in the gulf of guinea.
+the Web Mercator projection, the coordinate is right in the gulf of guinea.
```

Comment:
You fixed the case for "Africa". What's your opinion on "gulf of guinea"? Also worth fixing?

Reply:
Must be fixed too :  Gulf of Guinea